src/tests/ftest/nvme/pool_exclude.py

```diff
@@ -27,15 +32,65 @@ def setUp(self):
         super().setUp()
         self.dmg_command = self.get_dmg_command()
         self.daos_command = self.get_daos_command()
-        self.ior_test_sequence = self.params.get("ior_test_sequence",
-                                                 '/run/ior/iorflags/*')
+        self.ior_test_sequence = self.params.get("ior_test_sequence", "/run/ior/iorflags/*")
         # Recreate the client hostfile without slots defined
         self.hostfile_clients = write_host_file(
             self.hostlist_clients, self.workdir, None)
         self.pool = None
         self.cont_list = []
         self.dmg_command.exit_status_exception = True
 
+    def thread_run_ior(self, thread_queue, job_id, test, manager, log, hosts, path, slots, group,
```

Comment:
This should be defined in ftest/util/ior_utils.py so that it is available for more tests.